src/components/common/PermissionGroup.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <section class="permission-group">
+    <p class="permission-group-title">
+      {{ scopesTitle }}
+    </p>
+    <p
+      v-if="!isScopesEmpty"
+      class="permission-group-scopes"
+    >
+      {{ $t('components.scopes.access') }} {{ scopesDescription }}
+    </p>
+  </section>
+</template>
+
+<script>
+import camelCase from 'lodash/camelCase';
+import { SCOPES_TRANSLATES } from '@/constants/translates';
+
+export default {
+  name: 'PermissionGroup',
+
+  props: {
+    rootScope: {
+      type: String,
+      required: true,
+    },
+
+    scopes: {
+      type: Array,
+      default: () => [],
+    },
+  },
+
+  computed: {
+    isScopesEmpty() {
+      return this.scopes.length === 0;
+    },
+
+    scopesTitle() {
+      const normalizedKey = this.normalizeScopeKey(this.rootScope);
+
+      return SCOPES_TRANSLATES[normalizedKey] || normalizedKey;
```

Comment:
looks strange, we return translated text or key.
when transtale cannot be?

Reply:
If we will receive new, unhandled scope

Reply:
if we receive new unhandled scope, we should make error and send to sentry to fix it.
but showing raw data for user is not good point, as for me.